Slambook/index.js

```diff
@@ -1,14 +1,26 @@
 let form;
 
 let allSlamArray=[];
+let formValue = {};
```

Comment:
Redundant assignment of an empty object to 'formValue' at line 4, remove this line as it is already declared at line 1.